### ISSUE TITLE
Dropdown not functioning correctly for navbar

### ISSUE THREAD

**Issue by SylentNight37** [uikit/uikit#2433]
I'm having an issue with dropdowns disappearing when using it in conjunction with a navbar when configured with a hyperlink as the trigger. When I configure a dropdown with a button, it appears to work okay. The navbar examples for the dropdown triggered by a hyperlink in the documentation do not appear to work correctly.

I copied the left-nav part of the navbar with dropdowns section of the navbar documentation to make sure that I didn't typo anything. With this code, the dropdown appears, but disappears when I try to mouse over.

```
<nav class="uk-navbar-container" uk-navbar>
    <div class="uk-navbar-left">
        <ul class="uk-navbar-nav">
            <li class="uk-active"><a href="#">Active</a></li>
            <li>
                <a href="#">Parent</a>
                <div class="uk-navbar-dropdown">
                    <ul class="uk-nav uk-navbar-dropdown-nav">
                        <li class="uk-active"><a href="#">Active</a></li>
                        <li><a href="#">Item</a></li>
                        <li class="uk-nav-header">Header</li>
                        <li><a href="#">Item</a></li>
                        <li><a href="#">Item</a></li>
                        <li class="uk-nav-divider"></li>
                        <li><a href="#">Item</a></li>
                    </ul>
                </div>
            </li>
            <li><a href="#">Item</a></li>
        </ul>
    </div>
</nav>
```

If I replace the hyperlink with a button, it works correctly, and I'm able to mouse over the dropdown to click on one of the menu items:

```
<nav class="uk-navbar-container" uk-navbar>
      <div class="uk-navbar-left">
          <ul class="uk-navbar-nav">
              <li class="uk-active"><a href="#">Active</a></li>
              <li>
                  <button class="uk-button uk-button-default" type="button">Hover</button>
                  <div class="uk-navbar-dropdown">
                      <ul class="uk-nav uk-navbar-dropdown-nav">
                          <li class="uk-active"><a href="#">Active</a></li>
                          <li><a href="#">Item</a></li>
                          <li class="uk-nav-header">Header</li>
                          <li><a href="#">Item</a></li>
                          <li><a href="#">Item</a></li>
                          <li class="uk-nav-divider"></li>
                          <li><a href="#">Item</a></li>
                      </ul>
                  </div>
              </li>
              <li><a href="#">Item</a></li>
          </ul>
      </div>
    </nav>
```

**Comment by janschoenherr** [comment]
Could you please show an example of the issue in codepen.io. Thanks.

**Comment by SylentNight37** [comment]
Sure thing:

[http://codepen.io/SylentNight/pen/JEQVyb](url)

**Comment by SylentNight37** [comment]
I added a nav-bar right to the codepen I linked to above to try to demonstrate it working with using a button instead of a hyperlink. Interestingly, the way it's coded works for the dropdown when using the button as the trigger. However, if you remove the li after the dropdown (<li><a href="#">Item</a></li>), it works the same as when using a hyperlink as the trigger.

It appears that the hyperlink as the trigger may not be the issue, but instead is something with the way the dropdown is being triggered from the navbar?

**Comment by zzseba78** [comment]
It works perfect for me, in Firefox, Chrome, Safari.

**Comment by SylentNight37** [comment]
Yes, I've tried all the same browsers, and am experiencing the issue described above.

I did find that adding uk-dropdown="pos: bottom-center" to the div with uk-navbar-dropdown allows the dropdown to persist when mouseover.

**Comment by zzseba78** [comment]
Ok, and if you change the uikit.js version to beta 11 or 10 it works fine right?

**Comment by SylentNight37** [comment]
Yes, reverting the uikit.js to v11 makes it work as expected.

**Comment by zzseba78** [comment]
Seems like a bug...

**Comment by SylentNight37** [comment]
Agreed...v12 is the first version of UIKit 3 that I've worked with. I reverted my project to us v11 for now to continue development. Hopefully they get this worked out for the next version.

**Comment by zzseba78** [comment]
Sure, they will fix it

**Comment by SylentNight37** [comment]
Yeah, I've been impressed with the quality of the components that they've built for the framework. I can't wait for a stable release of UIKit 3.

**Comment by rustyd0g** [comment]
Same issue for me in beta 12.

[http://codepen.io/anon/pen/JEgoBW](http://codepen.io/anon/pen/JEgoBW)

**Comment by janschoenherr** [comment]
Fixed with b0ca0d6c19a57a6d8cad33dee61850919876bea0. Thanks
